unity-client/Assets/Scripts/MainScripts/DCL/Controllers/CharacterController/FreeMovementController.cs

```diff
@@ -0,0 +1,99 @@
+using DCL.Helpers;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class FreeMovementController : MonoBehaviour
+{
+
+    [Header("Movement")]
+    public float groundCheckExtraDistance = 0.25f;
+    public float movementSpeed = 11f;
+    public float runningSpeedMultiplier = 0.36f;
+
+    [Header("InputActions")]
+    public InputAction_Hold sprintAction;
+
+    [System.NonSerialized] public CharacterController characterController;
+
+
+
+    bool isSprinting = false;
+    bool isActive;
+
+    float deltaTime = 0.032f;
```

Comment:
Why are we using a fixed delta time and not `Time.deltaTime`? It should be converted to a local const in case we keep it

Reply:
I don't like it either, I just copy and paste some of the code of `DCLCharacterController` to maintain the same structure while moving the character.

You have this in `DCLCharacterController`
`    float deltaTime = 0.032f;
    float deltaTimeCap = 0.032f; // 32 milliseconds = 30FPS, 16 millisecodns = 60FPS`

I don't know why you use it that way, but I didn't want to change something that was working and in order to maintain a similar approach, I just copied and pasted. Let me know if you want it in a different way

Reply:
Okay! let's keep an eye on it in case is not needed, but after this pr, of course!